modules/kubernetes/DEVNOTES.txt

```diff
@@ -23,7 +22,7 @@ Once this is done, make sure that Minikube sees Docker images registered locally
     eval $(minikube docker-env)
 
 Start Minikube:
-    minikube start --vm-driver=xhyve
+    minikube start
```

Comment:
What's your operating system? It was required to pass in "--vm-driver=xhyve" parameter on my Mac OS laptop. Probably, we need to mention that this parameter has to be added on that operating system.

Reply:
I use MacOS too, default provider works fine